OGP1617-Asteroids/src/asteroids/model/programs/expressions/Any.java

```diff
@@ -0,0 +1,28 @@
+package asteroids.model.programs.expressions;
+
+import asteroids.model.Entity;
+
+public class Any extends EntityLiteral{
+
```

Comment:
Een 'Any' type is niet nodig daar maken we een expression voor die als resultaat een EntityLiteral geeft